tasks/gitlab_helpers.py

```diff
@@ -0,0 +1,78 @@
+"""
+Helper for generating links to CI Visibility for Gitlab CI jobs
+"""
+
+import json
+import os
+from urllib.parse import quote
+
+from invoke import task
+
+CI_VISIBILITY_URL = "https://app.datadoghq.com/ci/pipeline-executions"
+
+
+@task
+def generate_ci_visibility_links(_ctx, output: str | None):
+    """
+    Generate links to CI Visibility for the current job
+    usage deva gitlab.generate-ci-visibility-links
+    Generated file
+    """
+    ci_job_id = os.environ.get("CI_JOB_ID")
+    if ci_job_id is None:
```

Comment:
```suggestion
    if not ci_job_id:
```

More pythonic and would also catch empty strings